wgpu-core/src/device/global.rs

```diff
@@ -2197,7 +1819,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                     unreachable!("Fallback system failed to choose present mode. This is a bug. Mode: {:?}, Options: {:?}", config.present_mode, &caps.present_modes);
                 };
 
-                log::info!(
+                log::warn!(
```

Comment:
why are these changed to warning? Wouldn't that imply that using `Auto` is an error?